model/jpa/testData/create_db_content.ddl

```diff
@@ -71,7 +71,7 @@ CREATE TABLE push_message_info (
   client_identifier  VARCHAR(255)  DEFAULT NULL,
   ip_address         VARCHAR(255)  DEFAULT NULL,
   push_application_id VARCHAR(255) NOT NULL,
-  raw_json_message    VARCHAR(4500) DEFAULT NULL,
+  raw_json_message    LONGTEXT DEFAULT NULL,
```

Comment:
@sinarz I think this causing errors with Derby.. (that's used in here)

In the migration folder we actually do use embedded MySQL - perhaps worth to do the same here ?